code/modules/clothing/suits/marine_armor.dm

```diff
@@ -283,11 +283,11 @@
 	name = "\improper B18 defensive armor"
 	desc = "A heavy, rugged set of armor plates for when you really, really need to not die horribly. Slows you down though.\nHas an automated diagnostics and medical system for keeping its wearer alive."
 	icon_state = "xarmor"
-	soft_armor = list("melee" = 65, "bullet" = 75, "laser" = 75, "energy" = 80, "bomb" = 80, "bio" = 65, "rad" = 20, "fire" = 80, "acid" = 65)
+	soft_armor = list("melee" = 75, "bullet" = 75, "laser" = 75, "energy" = 80, "bomb" = 80, "bio" = 65, "rad" = 20, "fire" = 80, "acid" = 65)
```

Comment:
i don't know how good this actually is tbh, this makes 80 armor with the jumpsuit, might want to buff both it and b17 to 70 ?, even more counting both have same slowdown now

Reply:
i buffed it up cause you pay almost 2x more for B18 than B17; but if thats too much i can up the price or lower it back down to 70 on chest instead

which ones a better solution?

Reply:
you can put down the price, i believe, too much costly also means its less used overall

Reply:
lowered to 75 chest and helmet armor on par with B17, keeping price to 120 instead of 130 to compensate for autodoc of B18